x-pack/plugins/apm/server/lib/transaction_groups/get_error_rate.ts

```diff
@@ -183,7 +183,7 @@ export async function getErrorRatePeriods({
     previousPeriodPromise,
   ]);
 
-  const firtCurrentPeriod = currentPeriod.transactionErrorRate.length
+  const firstCurrentPeriod = currentPeriod.transactionErrorRate.length
     ? currentPeriod.transactionErrorRate
     : undefined;
```

Comment:
Not necessary, but since you've already refactored it a bit ☺️
```suggestion
  const firstCurrentPeriod = currentPeriod?.transactionErrorRate;
```

Reply:
Good catch. Actually it can be changed to:
```suggestion
  const firstCurrentPeriod = currentPeriod.transactionErrorRate
```